tensorflow_lite_support/python/task/processor/proto/segmentations_pb2.py

```diff
@@ -13,7 +13,207 @@
 # limitations under the License.
 """Segmentations protobuf."""
 
+import dataclasses
+from typing import Any, Tuple, List
+
+from tensorflow.tools.docs import doc_controls
 from tensorflow_lite_support.cc.task.vision.proto import segmentations_pb2
 
-Segmentation = segmentations_pb2.Segmentation
-SegmentationResult = segmentations_pb2.SegmentationResult
+_Segmentation = segmentations_pb2.Segmentation
```

Comment:
ditto. Please append `Proto` affix to proto class names to make it clearer that they're protobuf classes.